### PR TITLE
Updated CLion Version To 2019.3 And CLion Bazel Plugin To 2020.04.13

### PR DESCRIPTION
**Pull request by garethellis0**
### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [ ] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

**Review by akhilveeraghanta — APPROVED** [review]
LGTM :shipit:

**Comment by garethellis0** [comment]
Definitely some reasonable performance improvements in this CLion version, certainly startup is a bit quicker.

**Review by MathewMacDougall — APPROVED** [review]

**Review by jonathanlew — APPROVED** [review]

File: environment_setup/install_clion.sh [hunk 1]
```diff
@@ -42,10 +42,10 @@ echo "================================================================"
 echo "Installing Bazel Plugin"
 echo "================================================================"
 
-clion_plugin_dir="${HOME}/.CLion${clion_version_year}.${clion_version_major}/config/plugins"
-bazel_plugin_version_year="2019"
-bazel_plugin_version_major="12"
-bazel_plugin_version_minor="16"
+clion_plugin_dir="${HOME}/.CLion${clion_version_year}.{${clion_version_major}}/config/plugins"
```
Comment (MathewMacDougall) [inline environment_setup/install_clion.sh:45]: @garethellis0 didn't notice this until I ran locally. Why add the extra `{}`? Not it shows up in the filename which seems odd

`~/.CLion2019.{3}`

Comment (garethellis0) [reply environment_setup/install_clion.sh:45]: It showed up in the filename for me for some reason.

**Pull request merged**